OrbitQt/main.cpp

```diff
@@ -253,8 +253,14 @@ int main(int argc, char* argv[]) {
 #endif
 
     QApplication app(argc, argv);
-    QCoreApplication::setApplicationName("Orbit Profiler [BETA]");
-    QCoreApplication::setApplicationVersion(QString::fromStdString(OrbitCore::GetVersion()));
+    QApplication::setApplicationName("orbitprofiler");
+
+    // The application display name is automatically appended to all window titles when shown in the
+    // title bar: <specific window title> - <application display name>
```

Comment:
So if the window title is empty the " - " is not there, right?

Reply:
Exactly!